### PR TITLE
Create what is JavaScript

### PR DESCRIPTION
**Pull request by aman6969**
Part of #65

### CODE REVIEW

File: what is java script [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+Later in 1995, as Microsoft rise-up to the competition with the threat the web created, the project Internet Explorer started in an all-out attempt to wrestle control of the emerging platform from Netscape.
+
+Gradually Microsoft became a mortal threat, compelling Netscape with its Internet Explorer. Slowly, a standardization process started to develop to prevent Microsoft from gaining the power of the JavaScript language. Furthermore, they partnered with Sun to influence their shared interest in breaking the Microsoft monopoly.
+
+Brendan Eich has assumed that Sun on board decided to surf the tidal wave of building up surrounding Java and position JavaScript as the companion language with Java, in a similar manner, Visual Basic was to C++. Netscape's Mocha was later named JavaScript and aimed to turn the web into a full-blown application platform.
+
+Unfortunately for JavaScript, its early marketing position outlasted its usefulness and later became a mark on market acceptance because it emerged as a viable technology.
```
Comment (bpires) [inline what is java script:26]:
```suggestion
```

Direct copy of [w3schools: history of JavaScript](https://www.w3schools.in/javascript/history-of-javascript)

**Review by bpires — COMMENTED** [review]
Hi @aman6969, welcome to the community! Thanks so much for your interest in contributing!

I'm afraid that we have several issues to address here:

1. The file created must be a markdown file. That means it has to be of .md extension, as well as follows the markdown specs.
2. Your .md file must be created in the correct place within the folder structure, among the other topics, and follow naming convention.
3. Your text must be organized in sections, divided by heading levels in a hierarchical order.
4. The links must use the Markdown syntax and, also, you should link to the direct url, and not the bing search one.
5. Please read the [project charter](https://github.com/The-Mycelium-Network/webworkbook#what-are-we-trying-to-achieve) to understand better what we are trying to achieve. The idea here is not to do an extensive write-up about the topics, but to give simple explanations and notes, pointing to more in-depth resources. When writing content, try to focus on writing a synopsis and linking to the content for further reading.
6. On that note, you can't use other texts verbatim (word to word), specially without quotes and proper reference. That is plagiarism and is against our [Contributing Guidelines](https://github.com/The-Mycelium-Network/webworkbook/blob/main/CONTRIBUTING.md).
7. For videos, you should use [Hugo shortcodes](gohugo.io/content-management/shortcodes/#use-hugos-built-in-shortcodes) to embed the video in context.

You can check [The Markdown Guide](https://www.markdownguide.org/) for a quick start on this mark-up language and learn its [basic syntax here](https://www.markdownguide.org/basic-syntax/).

File: what is java script [hunk 2]
```diff
@@ -0,0 +1,113 @@
+What is JavaScript?
+JavaScript was initially created to “make web pages alive”.
+
+The programs in this language are called scripts. They can be written right in a web page’s HTML and run automatically as the page loads.
+
+Scripts are provided and executed as plain text. They don’t need special preparation or compilation to run.
+
+In this aspect, JavaScript is very different from another language called Java.
```
Comment (bpires) [inline what is java script:8]: 🚫 Direct copy of [javascript.info](https://javascript.info/intro)

File: what is java script [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+JavaScript is a lightweight programming language that web developers commonly use to create more dynamic interactions when developing web pages, applications, servers, and or even games.
+
+Developers generally use JavaScript alongside HTML and CSS The scripting language works well with CSS in formatting HTML elements. However, JavaScript still maintains user interaction, something that CSS cannot do by itself.
+
+JavaScript’s implementations within the web, mobile application, and game development make the scripting language worth learning. You can do so via learning platforms like BitDegree or by exploring free JavaScript templates and applications on code hosting platforms like GitHub.
```
Comment (bpires) [inline what is java script:13]: 🚫 Direct copy of [hostinger.com](https://www.hostinger.com/tutorials/what-is-javascript#:~:text=JavaScript%20is%20a%20lightweight%20programming,CSS%20in%20formatting%20HTML%20elements.)

File: what is java script [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+Symbol ‒ provides unique identifiers for objects.
+Object ‒ for complex data structures written with curly braces. For example, {item:”Book”, information:”biography”}.
+Primitive data types, consisting of all data types except object, can only store a single data. Meanwhile, the object data type can contain a collection of values.
+
+With JavaScript, you can also improve users’ web browsing experience using cookies. Creating, reading, and deleting cookies in JavaScript requires the document.cookie property, which functions as the getter and setter of the cookies’ values.
+
+4. Game Development
+JavaScript can help you build a game if used with HTML5 and an Application Programming Interface (API) like WebGL. Plenty of JavaScript-based game engines like Phaser, GDevelop, and Kiwi.js are available for rendering graphics, code recycling, and cross-platform applications.
+
+Some of the games made with JavaScript game engines include Angry Birds, The Wizard, and 2048.
```
Comment (bpires) [inline what is java script:83]: 🚫 Direct copy of [hostinger.com](https://www.hostinger.com/tutorials/what-is-javascript#:~:text=JavaScript%20is%20a%20lightweight%20programming,CSS%20in%20formatting%20HTML%20elements.)

File: what is java script [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+ 
+STANDARDS
+The beauty of JavaScript Standard Style is that it's simple. No one wants to maintain multiple hundred-line style configuration files for every module/project they work on. Enough of this madness!
+
+This module saves you (and others!) time in three ways:
+
+No configuration. The easiest way to enforce consistent style in your project. Just drop it in.
+Automatically format code. Just run standard --fix and say goodbye to messy or inconsistent code.
+Catch style issues & programmer errors early. Save precious code review time by eliminating back-and-forth between reviewer & contributor.
+Adopting standard style means ranking the importance of code clarity and community conventions higher than personal style. This might not make sense for 100% of projects and development cultures, however open source can be a hostile place for newbies. Setting up clear, automated contributor expectations makes a project healthier.
```
Comment (bpires) [inline what is java script:93]: 🚫 Direct copy of [standardjs.com](https://standardjs.com/)

**Comment by bpires** [comment]
Looks like all the text was copied from other sources, and you haven't added any original text. That goes against our contributing guidelines and configures straight-up plagiarism. For that reason, I'm closing this PR.

You should double-check [our guidelines](https://github.com/The-Mycelium-Network/webworkbook/blob/main/CONTRIBUTING.md) and our [project charter](https://github.com/The-Mycelium-Network/webworkbook) to understand our goals and values.

> We are open-source enthusiasts! 🌱 Feel free to get contribute to any aspect of this project. While doing so, be reasonable: do not just copy content from elsewhere. Like academic writing, your work can incorporate the ideas of others but should reflect your original approach to the problem.

Also, please take some time to explore the repo before proposing a PR to make sure you are going along accordingly with the work that has already been put in place.

Please don't be discouraged. I'll be pleased to review a new PR at any time. I've left some notes and comments for you on this PR with issues you have to address. Please, if you need further assistance, ask for help from our [community on Discord](https://github.com/The-Mycelium-Network/the-mycelium-network).

**Comment by schalkneethling** [comment]
Thank you so much for your detailed review and feedback here, @bpires. @aman6969 I have to say I am a bit disappointed. It is best to ask questions about the issue if you are uncertain about anything before opening a pull request.

I also linked to the [contribution guidelines](https://github.com/The-Mycelium-Network/webworkbook/blob/main/CONTRIBUTING.md) in the issue, so I would appreciate it if you took some time to read it. If anything is unclear, please ask questions. We are always happy to help.

We do have some shortcomings in our documentation, but as a new project, we are learning as we go. Asking questions is a part of that learning experience for everyone involved. If you are still interested in working on this issue, please open a new pull request, but before you do, please be sure that you have a clear understanding of what we are trying to achieve with this project, how the content fits together, and how the platform ([Hugo](https://gohugo.io)) works.

Again, if anything is unclear and you have questions, please post those on the issue, and we can help you there.

**Pull request closed without merging**